examples/experimental/i3s-17-and-debug/normals-utils.js

```diff
@@ -0,0 +1,59 @@
+import {Vector3} from 'math.gl';
+import {Ellipsoid} from '@math.gl/geospatial';
+
+const ADDITIONAL_SCALE = 200; // scaling to make normals visible on the map.
+
+export function generateBinaryNormalsDebugData(tile) {
+  if (
+    !tile.content ||
+    !tile.content.attributes ||
+    !tile.content.attributes.normals ||
+    !tile.content.attributes.positions
+  ) {
+    return {};
+  }
+
+  const normals = tile.content.attributes.normals.value;
+  const positions = tile.content.attributes.positions.value;
+
+  return {src: {normals, positions}, length: positions.length};
+}
+
+export function getNormalSourcePosition(index, data, normalsGap) {
+  let sourcePosition = {};
+
+  if (index % normalsGap === 0 || normalsGap === 0) {
+    const positions = data.src.positions;
+    const position = new Vector3([
+      positions[index * 3],
+      positions[index * 3 + 1],
+      positions[index * 3 + 2]
+    ]);
+    sourcePosition = Ellipsoid.WGS84.cartesianToCartographic(position);
+  }
+
+  return sourcePosition;
+}
+
+export function getNormalTargetPosition(index, data, normalsGap) {
```

Comment:
Normally a minimal test case for every helper function

Reply:
Sure it will be good. But there are no any test cases in examples folder. As I know environment allows us to use tests inside module folder only.